stock_available_to_promise_release/models/stock_move.py

```diff
@@ -603,6 +614,81 @@ def _get_chained_moves_iterator(self, chain_field):
             yield moves
             moves = moves.mapped(chain_field)
 
+    def _return_quantity_in_stock(self, qty_to_return_per_move):
+        """Return a quantity from a list of moves.
+
+        The quantity to return is in the product uom"""
+        moves_to_return = self.browse([m_id for m_id in qty_to_return_per_move.keys()])
+        moves_per_type = groupby(moves_to_return, lambda m: m.picking_type_id)
+        for picking_type, moves_list in moves_per_type:
+            moves = self.browse().concat(*moves_list)
```

Comment:
It is better to use union, concat would list a same move twice. 
```suggestion
            moves = self.browse().union(*moves_list)
```